data/human/deep missions.txt

```diff
@@ -3374,10 +3373,11 @@ mission "Mani Refit"
 		not planet "Mani"
 	destination "Mani"
 	on offer
-		dialog
+		conversation
 			`Your inbox blinks with a new message from the Transport Union:`
-			`	The fleet is ready for its inspirational journey. Please return to Mani.`
-			`		-Jayden, Mani Administrator, Transport Union`
+			`	"The fleet is ready for its inspirational journey. Please return to Mani.`
+			`		-Jayden, Mani Administrator, Transport Union"`
```

Comment:
I don't think stuff like this usually gets quotes.
See the messages the player receives in "FW Commitment" and "FW Wolf Pack 2".
You could, however, instead present the message in-line, as is done in "Smuggler's Den: Payment".

Reply:
Those others should probably be adjusted as well then (either here or separately), the addition of quotes here is correct.

If we were -actually- presenting an image of the message as a cutscene instead (which I would not support stylistically, even if it didn't sound like a whole chore in itself), then it'd be proper to leave them off, but even with using spacing to mirror that concept, it's still quoting the message, not showing it.